.github/workflows/backend-lint.yml

```diff
@@ -62,9 +62,11 @@ jobs:
         if: steps.changes.outputs.backend == 'true'
         with:
           path: ${{ steps.pip.outputs.pip-cache-dir }}
-          key: ${{ runner.os }}-py${{ steps.python-version.outputs.python-version }}-pip${{ steps.pip.outputs.pip-version }}-${{ hashFiles('**/requirements-*.txt') }}
+          # Note this uses a different cache key than other backend-based workflows because this workflows dependencies are different
+          key: |
+            precommit-${{ runner.os }}-py${{ steps.python-version.outputs.python-version }}-pip${{ steps.pip.outputs.pip-version }}-${{ hashFiles('requirements-pre-commit.txt') }}
           restore-keys: |
-            ${{ runner.os }}-py${{ steps.python-version.outputs.python-version }}-pip${{ steps.pip.outputs.pip-version }}
+            precommit-${{ runner.os }}-py${{ steps.python-version.outputs.python-version }}-pip${{ steps.pip.outputs.pip-version }}
```

Comment:
No `secrets.PIP_CACHE_VERSION` portion here for the precommit caches?

Also, what's the reasoning behind that? I could invalidate these caches if I wanted to by like, inserting a temporary comment into the requirement files.